vcd/resource_vcd_network_isolated.go

```diff
@@ -157,15 +157,15 @@ func resourceVcdNetworkIsolatedCreate(d *schema.ResourceData, meta interface{})
 		Configuration: &types.NetworkConfiguration{
 			FenceMode: "isolated",
 			IPScopes: &types.IPScopes{
-				IPScope: types.IPScope{
+				IPScope: []*types.IPScope{&types.IPScope{
```

Comment:
Why are we making this an array if we're then using **only** the first occurrence just a few lines below?

Reply:
I needed to change to match govcd API